examples/oapp/test/foundry/MyOApp.t.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.20;
+
+// MyOApp imports
+import { MyOApp } from "../../contracts/MyOApp.sol";
+
+// OApp imports
+import { IOAppOptionsType3, EnforcedOptionParam } from "@layerzerolabs/lz-evm-oapp-v2/contracts/oapp/libs/OAppOptionsType3.sol";
+import { OptionsBuilder } from "@layerzerolabs/lz-evm-oapp-v2/contracts/oapp/libs/OptionsBuilder.sol";
+
+// OZ imports
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
+
+// Forge imports
+import "forge-std/console.sol";
+
+// DevTools imports
+import { TestHelperOz5 } from "@layerzerolabs/test-devtools-evm-foundry/contracts/TestHelperOz5.sol";
+
+contract MyOAppTest is TestHelperOz5 {
+    using OptionsBuilder for bytes;
+
+    uint32 aEid = 1;
```

Comment:
NIT:  add an access modifier (i.e., `internal`).

Reply:
I am just gonna merge this one as is if that's okay - it's blocking a bunch of PRs from getting through and I don't wanna be waiting for another pipeline to go through. Will fix in the next one, I still want to drop the `turbo.json` files we used to work around the `solc` `text file busy`